src/Microsoft.DotNet.Web.Spa.ProjectTemplates/content/Angular-CSharp/ClientApp/package.json

```diff
@@ -12,28 +12,30 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^6.0.0",
-    "@angular/common": "^6.0.0",
-    "@angular/compiler": "^6.0.0",
-    "@angular/core": "^6.0.0",
-    "@angular/forms": "^6.0.0",
-    "@angular/http": "^6.0.0",
-    "@angular/platform-browser": "^6.0.0",
-    "@angular/platform-browser-dynamic": "^6.0.0",
-    "@angular/router": "^6.0.0",
+    "@angular/animations": "6.1.10",
+    "@angular/common": "6.1.10",
+    "@angular/compiler": "6.1.10",
+    "@angular/core": "6.1.10",
+    "@angular/forms": "6.1.10",
+    "@angular/http": "6.1.10",
+    "@angular/platform-browser": "6.1.10",
+    "@angular/platform-browser-dynamic": "6.1.10",
+    "@angular/platform-server": "6.1.10",
+    "@angular/router": "6.1.10",
     "@nguniversal/module-map-ngfactory-loader": "6.0.0",
     "core-js": "^2.5.4",
     "rxjs": "^6.0.0",
     "zone.js": "^0.8.26",
     "aspnet-prerendering": "^3.0.1",
-    "bootstrap": "^4.1.3"
+    "bootstrap": "^4.1.3",
+    "jquery": "3.3.1",
```

Comment:
Is this a new dependency or is it here to force it back to a specific version?

Reply:
jquery and popper.js are both Peer dependencies requestetd by bootstrap that we weren't previously fulfilling.

```
npm WARN bootstrap@4.1.3 requires a peer of jquery@1.9.1 - 3 but none is installed. You must install peer dependencies yourself.
npm WARN bootstrap@4.1.3 requires a peer of popper.js@^1.14.3 but none is installed. You must install peer dependencies yourself.
```

Reply:
OK